src/components/dialog/dialog.tsx

```diff
@@ -749,7 +750,7 @@ export class ChDialog {
           ref={el => (this.#dialogRef = el)}
         >
           {this.showHeader && (
-            <div
+            <header
```

Comment:
Don't add the header tag since the component doesn't have an article, aside, nav or section tag.

https://html.spec.whatwg.org/#sectioning-content

```suggestion
            <div
```